src/Components/SmartComponents/SystemsExposedTable/SystemsExposedTable.js

```diff
@@ -261,6 +261,7 @@ class SystemsExposedTable extends Component {
             };
             fetchData(cve, config);
             this.createUrlParams(config);
+            this.setState({ ...config });
```

Comment:
@mkholjuraev setting the state it will cause a re-render. Please make sure if this is wanted. Cause recently we removed `setState`  to solve this issue https://github.com/RedHatInsights/vulnerability-ui/commit/b4fa8c50b6f4ad1892eadf1a78e31cac106511cb#diff-54887a915157469cb0acca5eaeee7e7e